app/views/hq/admin_users/edit.html.erb

```diff
@@ -0,0 +1,22 @@
+<h1> Edit Admin User </h1>
+
+<%= form_for [:hq, @admin_user], :builder => OsraFormBuilder do |f| %>
+  <%= render 'shared/errors', object: f.object %>
+  <div class="form-group">
+    <%= f.label_for_field :email %>
+    <%= f.text_field :email, class: "form-control" %>
+  </div>
+  <div class="form-group">
+    <%= f.label_for_field :password %>
```

Comment:
That will change the text of the label to New Password ok. But the red "(required)" will still appear after it, because that's done in CSS styling based on the associated field being mandatory in the model.